album_art.py

```diff
@@ -0,0 +1,84 @@
+import requests
+import os
+import string
+import json
+from bs4 import BeautifulSoup
+
+def fetch(song, **kwargs):
+    if kwargs['backend'] == 'lastfm':
+        image = fetch_from_lastfm(song)
+    elif kwargs['backend'] == 'google':
+        image = fetch_from_google(song)
+    return image
+
+def fetch_from_google(song):
+    ''' Downloads art from images.google.com '''
+
+    query = song['artist'] + song['name'] + 'cover art google play' # Art work can be of huge size so the 'google play' in query mostly results into smaller images
```

Comment:
I don't know on what songs you tested this query on, but when I am testing this it is not showing good results.
Use `'%s %s' % (song['album'], song['artist'])`.